### ISSUE TITLE
Setup discovery for asterisk_mbox

### ISSUE THREAD

**Issue by felipecrs** [TECH7Fox/Asterisk-add-on#94]
The title says all, but I don't know if asterisk_mbox integration already supports receiving discovery information.

**Comment by TECH7Fox** [comment]
No, the integration doesn't even have a config flow. So if we want I guess we have to implement our own.

Don't think this should have much priority, maybe later. Will keep this open.

**Comment by felipecrs** [comment]
Right. Definitely not priority.
